src/org/thoughtcrime/securesms/ProfileDocumentsFragment.java

```diff
@@ -160,7 +160,6 @@ private void handleMediaPreviewClick(@NonNull DcMsg dcMsg) {
   public void onMediaLongClicked(DcMsg mediaRecord) {
     if (actionMode == null) {
       ((ProfileDocumentsAdapter) recyclerView.getAdapter()).toggleSelection(mediaRecord);
-      recyclerView.getAdapter().notifyDataSetChanged();
```

Comment:
this seems unnecessary, `notifyDataSetChanged()` is called inside `toggleSelection()` called in the previous line